pennylane/templates/utils.py

```diff
@@ -116,54 +108,50 @@ def _check_shape(inpt, target_shape, msg=None, bound=None):
     return shape
 
 
-def _check_shapes(inpt_list, target_shape_list, bound_list=None, msg=None):
-    """Same as _check_shape, but for lists.
+def _check_shapes(inpt_list, target_shapes, msg, bounds=None):
+    """Same as `_check_shape()`, but for lists of elements.
```

Comment:
I would consider to copy-paste the description of `_check_shape` so that the docstring of this function is self-contained.